packages/kotti-ui/source/kotti-field-select/components/GenericSelectField.vue

```diff
@@ -154,6 +160,16 @@ export default defineComponent({
 		const { isDropdownOpen, isDropdownMounted, ...selectTippy } =
 			useSelectTippy(field)
 
+		onMounted(() => {
+			const fieldInputContainer = document.getElementById(
+				`${field.inputProps.id}-input-container`,
+			)
+
+			if (!fieldInputContainer) return
+
+			selectTippy.setTriggerTarget(fieldInputContainer)
```

Comment:
Cannot do that because
1. fieldInputContainer doesn't live here but in `KtField`
2. when `useTippy` runs, the components haven't mounted yet so the references are undefined